### DISCUSSION TITLE
Maths documentation

### DISCUSSION THREAD

**Discussion by mildfuzz** [pixijs/pixijs#11097] [Q&A] [answered]
Maybe I am going mad but, I stumbled across this Class https://github.com/pixijs/pixijs/blob/dev/src/maths/shapes/Circle.ts

and I can't find any documentation around any of the classes in the maths folder. Am I missing something?

**Comment by srpatel** [comment, marked as answer]
Docs for the maths folder are here: https://pixijs.download/release/docs/maths.html
Or, specifically, the Circle class: https://pixijs.download/release/docs/maths.Circle.html
